### PR TITLE
Remove !setinfo and !unsetinfo commands

### PR DESCRIPTION
**Pull request by jmariondev**
These have been made mostly redundant by Discord's integration with gaming services and are mostly used nowadays for people to store useless information about themselves that nobody will look at.

per @Reynbow @ThomasShaped

### CODE REVIEW

**Review by otter-computer — APPROVED** [review]
I love seeing this in red and want them to go away thx, but will wait for @brandon-a 's approval also 😬

**Review by brandon-a — APPROVED** [review]

**Pull request merged**
